angular/src/lib/alert-banner/alert-banner.component.ts

```diff
@@ -41,9 +41,11 @@ export class AlertBannerComponent implements OnInit {
   @Output() hide: EventEmitter<any> = new EventEmitter();
 
   @HostListener('document:keydown.escape', ['$event']) handleKeydown(
-    _: KeyboardEvent
+    event: KeyboardEvent
   ) {
-    this.hide.emit();
+    if (event.key !== 'Tab') {
+      this.hide.emit();
+    }
```

Comment:
I'm confused .. we want escape and tab to close the alert banner? or we don't want tab to close the alert banner?

Reply:
We don't want tab to close the alert banner

Reply:
thanks